src/controllers/OrdersController.php

```diff
@@ -315,7 +315,11 @@ public function actionCompleteOrder(): Response
 
         if ($order && !$order->isCompleted && $order->markAsComplete()) {
             $date = new \DateTime($order->dateOrdered);
-            return $this->asJson(['success' => true, 'dateOrdered' => $date]);
+            return $this->asJson([
+                'success' => true,
+                'dateOrdered' => $date,
+                'order' => $order,
```

Comment:
I noticed that the order should be passed through `BaseFrontEndController::cartArray()` before being made public in this case.